main.tf

```
@@ -112,6 +112,16 @@ resource "aws_lambda_function" "this" {
     }
   }
 
+  dynamic "logging_config" {
+    for_each = var.logging_config_log_format != null ? [true] : []
+    content {
+      log_format            = var.logging_config_log_format
```

```suggestion
      log_format            = var.logging_log_format
```

Let's drop `config_` from variable names like we do in other places.